files/orchestrate/README.md

```diff
@@ -7,22 +7,22 @@
 3. [Dev Network Setups](#dev-network-setups)
    1. [Deploy contracts & send transactions](#orchestrate-contracts)
    2. [Multitenancy](#orchestrate-multitenancy)
-                  
+
 
 ## Prerequisites
 
 To run these tutorials, you must have the following installed:
 
 - [Docker and Docker-compose](https://docs.docker.com/compose/install/)
-
+f
 | ⚠️ **Note**: If on MacOS or Windows, please ensure that you allow docker to use upto 4G of memory or 6G if running Privacy examples under the _Resources_ section. The [Docker for Mac](https://docs.docker.com/docker-for-mac/) and [Docker Desktop](https://docs.docker.com/docker-for-windows/) sites have details on how to do this at the "Resources" heading       |
-| ---                                                                                                                                                                                                                                                                                                                                                                                |
+| ---
 
 
-| ⚠️ **Note**: This has only been tested on Windows 10 Build 18362 and Docker >= 17.12.2                                                                                                                                                                                                                                                                                              |
-| ---                                                                                                                                                                                                                                                                                                                                                                                |
+| ⚠️ **Note**: This has only been tested on Windows 10 Build 18362 and Docker >= 17.12.2
+| ---
 
-- On Windows ensure that the drive that this repo is cloned onto is a "Shared Drive" with Docker Desktop
+- On Windows ensure that the drive, that this repo is cloned onto, is a "Shared Drive" with Docker Desktop
 - On Windows we recommend running all commands from GitBash
```

Comment:
I think this recommendation should change to WSL2 w/ Docker Desktop installed such that it uses the WSL2 execution engine. I use it for testing all of this stuff, and it's a lot more reliable and easier to set up than other alternatives.

Reply:
Also that'll bring this README in line with the rest of the quickstart

Reply:
We aren't Windows users so happy to go with your suggestion. Let's treat that in another MR?